dashboard/modules/reporter/reporter_agent.py

```diff
@@ -74,10 +74,27 @@ def __init__(self, dashboard_agent):
                     f"{self._dashboard_agent.node_id}"
         # A list of gauges to record and export metrics.
         self._gauges = {
-            "node_cpu": Gauge("node_cpu", "Total CPU usage on a ray node",
-                              "percentage", ["ip"]),
-            "node_mem": Gauge("node_mem", "Total memory usage on a ray node",
-                              "bytes", ["ip"]),
+            "node_cpu_utilization": Gauge("node_cpu",
+                                          "Total CPU usage on a ray node",
+                                          "percentage", ["ip"]),
+            "node_cpu_count": Gauge("node_cpu_count",
```

Comment:
Is this necessary? Isn't it always a fixed value?

Reply:
not sure if it is always fixed but thought it would be good to show CPU available on the Grafana dashboard. @simon-mo ?

Reply:
good point. the issue with storing a fixed value as gauge is it will become an entire time series. 

maybe we should use something like Info:
https://github.com/prometheus/client_python#info
not sure if our exporter support it directly & whether or not grafana supports it. @kathryn-zhou can you look into this? We bundle the prometheus exporter in `ray/python/prometheus_exporter.py` so we can modify if needed.

Reply:
actually @rkooo567, @kathryn-zhou and I looked into info type and it is just a gauge. in order to show total number of nodes in grafana we need have to expose the value somehow and this seems like this most straightforward way to do it.

Reply:
I think my question here is isn't it enough to show CPU registered in GCS? Also, the number of CPU in each node is not something that should be metrics right? I feel like this is the information that. could be displayed in ray status instead. 

Is the usage of this metric always just summing it up to see the entire number of cpus spawned by autoscaler?

Reply:
Those are not equal. The gcs number should be the number of ray cpu resources which can be greater than the physical ones. (--num-cpus > of < actual cpus). The reporter here is reporting node metrics.